spec/controllers/courses_controller_spec.rb

```diff
@@ -60,10 +60,10 @@
   end
 
   describe "GET #index" do
-    it "returns a success response", :logged_lecturer do
+    it "returns a not successful response because /index shouldn't be accessed", :logged_lecturer do
       Course.create! valid_attributes
       get :index, params: {}, session: valid_session
-      expect(response).to be_successful
+      expect(response).not_to be_successful
     end
   end
 
```

Comment:
Please add a test that verifies that /courses links to the home view, as said in the AC